### ISSUE TITLE
Defining Critical JSON Schema Projects

### ISSUE THREAD

**Issue by benjagm** [json-schema-org/community#521]
### Objective
This issue aims to define what we understand by "critical" JSON Schema projects and establish the eligibility criteria and the support provided for these projects. Currently taking advantage of and supporting projects Bowtie that are not part of the Github Organization but are critical for the whole JSON Schema Ecosystem and this issue aims to standarize this situation by find a common definition, eligibility criteria and the benefits.

### Defining Critical Projects
To qualify as a "critical" project, a JSON Schema project should meet the following criteria:

* **Language Agnostic**: The project should be language-agnostic, ensuring its relevance to a wide range of developers working with JSON Schema, regardless of the programming language they are using.

* **Uniqueness**: The project should offer unique and valuable functionality, avoiding direct competition with existing tools in the JSON Schema ecosystem. It should fill a gap or provide an innovative solution to a problem not adequately addressed by other projects.

* **Impact**: Critical projects should have a significant and positive impact on the entire JSON Schema ecosystem, benefiting a large user base, or addressing key challenges faced by the community.

### Support for Critical Projects
The support provided to critical projects will include:

* **Participation in Mentoring and Contribution Programs**: Critical projects will be eligible for mentorship programs to help onboard new contributors, and they will have access to resources and guidance to foster their growth.

* **Promotion of New Features and Updates**: We will actively assist critical projects in promoting new features and updates to ensure they reach a wider audience, including cross-promotion within the JSON Schema community and on relevant platforms.

### Process to join the List
For a project to get this status there should be a TSC voting process.

### Current projects:

* [Bowtie](https://github.com/bowtie-json-schema/bowtie/)

**Prospect Candidates:** 
* [Learn JSON Schema](https://github.com/sourcemeta/learnjsonschema.com)
* [Alter Schema](https://github.com/sourcemeta/alterschema)

### Help us
We value your input and invite you to participate in this discussion. Please provide your feedback, suggestions, and nominations for projects that you believe meet the criteria for critical JSON Schema projects.

**Comment by benjagm** [comment]
@Relequestual @Julian @jviotti I'd love to get your thoughts here.

**Comment by Julian** [comment]
Alterschema is available as a CLI though no? And that's presumably the primary way people would use it at the minute? Or do you expect they use it as a library?

Because if it's the former, which would be my guess, then I think it's fine -- every tool of course is written in *some* language, Bowtie included -- but it's usable from any via the CLI. Of course if you use it in the host language maybe you get extra stuff (e.g. calling Bowtie from Python or whatever maybe is easier), but the spirit of language-agnostic is certainly there.

**Comment by benjagm** [comment]
> language agnostic web UI

In the issue I just provided an initial proposal to start the discussions. Regarding providing a web UI or even a CLI tool for my will be enough.

**Comment by jviotti** [comment]
Yeah, that's what I mean. If we clarify CLI / Web is an acceptable interface, then that sounds good to me, but I think it should be more clearly outlined ^

**Comment by benjagm** [comment]
> find consensus on what the "critical projects" program

My idea with this issue is to find a common definition of things... not to define the program itself. The program should have some strategic goals, KPIs and other elements that to be defined later, once we all have the same view and consensus to proceed.

**Comment by benjagm** [comment]
I don't see much progress. Is it ok If I proceed with a proposal of how this program will look like?

**Comment by Relequestual** [comment]
I'm now activly working on progressing this work.

**Comment by benjagm** [comment]
I am tagging this issue `do not close` to not be considered by the bot collecting stale issues.
